pkg/providers/common/common.go

```diff
@@ -30,18 +30,19 @@ func GetAuditPolicy() string {
 	return auditPolicy
 }
 
-func BootstrapClusterOpts(serverEndpoint string, clusterConfig *v1alpha1.Cluster) ([]bootstrapper.BootstrapClusterOption, error) {
+func BootstrapClusterOpts(clusterConfig *v1alpha1.Cluster, serverEndpoints ...string) ([]bootstrapper.BootstrapClusterOption, error) {
```

Comment:
Does it make sense for all providers? Having a serverEndpoints that will be added to NoProxy. Or is this specific to cloudstack? If so we might not wanna update this common method

Reply:
Currently, only cloudstack and vsphere use common.BootstrapClusterOpts and they both want to add server endpoint(s) to NoProxy. I think this makes sense for all providers.